<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="./">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>1 Library preparation &mdash; NGS-QC-Course  documentation</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=80d5e7a1" />
      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=19f00094" />

  
    <link rel="canonical" href="https://fabianandradelozano.github.io/NGS-QC-Course/1- Library preparation.html" />
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=5929fcd5"></script>
        <script src="_static/doctools.js?v=9a2dae69"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="author" title="About these documents" href="about.html" />
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="2 Main Sequencing Technologies" href="2-%20Sequencing%20technologies.html" />
    <link rel="prev" title="About the course" href="about.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            NGS-QC-Course
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="about.html">About the course</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">1 Library preparation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#main-causes-of-poor-quality-data">Main Causes of poor quality data</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#template-preparation">Template Preparation</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#id1">Library preparation</a></li>
<li class="toctree-l2"><a class="reference internal" href="#dna-library-bias">DNA library bias</a></li>
<li class="toctree-l2"><a class="reference internal" href="#rna-library-bias">RNA library bias</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#sample-preservation-and-isolation"><strong>Sample Preservation and Isolation</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="#library-construction"><strong>Library Construction</strong></a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="2-%20Sequencing%20technologies.html">2 Main Sequencing Technologies</a></li>
<li class="toctree-l1"><a class="reference internal" href="3-%20Quality%20Control%20and%20Preprocessing.html">3 Quality Control and Preprocessing</a></li>
<li class="toctree-l1"><a class="reference internal" href="4-%20Quality%20of%20the%20mapping.html">4 Quality of the Mapping</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">NGS-QC-Course</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">1 Library preparation</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/1- Library preparation.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="library-preparation">
<span id="library-preparation-page"></span><h1>1 Library preparation<a class="headerlink" href="#library-preparation" title="Link to this heading"></a></h1>
<section id="main-causes-of-poor-quality-data">
<h2>Main Causes of poor quality data<a class="headerlink" href="#main-causes-of-poor-quality-data" title="Link to this heading"></a></h2>
<p>Quality of the sequencing data starts before sequencing and library preparation. A proper Nucleic acid extractation is necessary to gurantee  a high quantity and purity.
Depending on the sample nature and the nucleic acid (RNA or DNA) the extraction proceess may vary, then an appropiate protocol to each situation must be chosen.
Thus, a rigorous quality control of the nucleic acid quantification extraction must to be performed, to asses the <strong>quantity, purity and integrity</strong>.
DNA Typically could be measured either UV spectrocopy (Nanodrop) or electrophoresis (Agilent TapeStation), RNA concentration is typically measured by Qubit fluorometer (ThermoFisherScientific).</p>
<div class="admonition danger">
<p class="admonition-title">Danger</p>
<p><strong>RNA is more critical</strong>, sample degradation and contamination is more frequent.</p>
</div>
<section id="template-preparation">
<h3>Template Preparation<a class="headerlink" href="#template-preparation" title="Link to this heading"></a></h3>
<p><em>Source: https://doi.org/10.1016/j.humimm.2021.02.012</em></p>
</section>
</section>
<section id="id1">
<h2>Library preparation<a class="headerlink" href="#id1" title="Link to this heading"></a></h2>
<p>Selecting  a suitable NGS library according to the type of sample (cell type or tissue), and the downstream analysis (WES, WGS, ChipSeq, RNA-seq, …) is essential to gurantee the quality of the data and get de desired information for our research.
In a nutshell, library is defined as a collection of nucleic acid (RNA or DNA) fragments of a defined lenght distribution with adapters attached.</p>
<p>Main steps of a Library Preparation Kit:</p>
<ul class="simple">
<li><p><strong>Fragmentation</strong></p></li>
</ul>
<p>Acid nucleic must be broken into smaller fragments. Generally physical and enzimatic fragmentation methods are popular for DNA,
whilst chemical fragmentation is more often applied to RNA fragmentation.</p>
<ul class="simple">
<li><p><strong>End-repair</strong></p></li>
</ul>
<p>With the use of enzymes (polymerases) are repaired  the 5’ and 3’ overhangs produced during the fragmentation, thus creating blunt ends.
Additionally, the 5’ ends are phosphorylated so they are compatible for the adapter ligation process. Some sequencing platforms (as Illumina) may also  require
the addition of a single adenine base to create a 3’ overhang before adapter ligation.</p>
<ul class="simple">
<li><p><strong>Adapter ligation</strong></p></li>
</ul>
<p>Short sequences of synthetic oligonucleotides (adapters) are attached to both ends of the DNA fragments.
These short sequences interact with oligonucleotides on the sequencing instrument’s flow cells to ensure that library is recognised and sequenced.
Adapters may also contain a barcode (short index) for multiplexing capabilities, This means that the fragments of the sample are identified when sequencing a pool of samples in the same flow cell.</p>
<ul class="simple">
<li><p><strong>Amplification</strong></p></li>
</ul>
<p>It is an optional step that enbales the sequencing library to be amplified via polymerase chain reaction (PCR), which allows lower samples inputs to be used for library preparation.
Can introduce <strong>GC bias, duplicates or artifacts</strong> that can hinder downstream analysis. In case a good quantity sample is preferable a PCR-free protocols to ensure high library complexity
and more readily enable specific applications (like Whole Genome Sequencing and SNP detection).</p>
<ul class="simple">
<li><p><strong>Purification</strong></p></li>
</ul>
<p>removal of unwanted products to leave only the nucleic acid fragments. Often is perfomed size selection by agarose gel or magnetic bead purification.</p>
<ul class="simple">
<li><p><strong>Quality control</strong></p></li>
</ul>
<p>Check if DNA mets the quantity and quality requirements of the sequencing instrument. Assesss the quantity and size distribution of the library.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>RNA library preparation is more complex due to the risk of degradation and requires additional steps respect DNA:</p>
<ul class="simple">
<li><p>Due that RNA is converted to cDNA, PCR-amplified libraries are necessary for many sequencing instruments.</p></li>
<li><p>Most of the RNA-seq applications requires the removal of the ribosomal RNA (rRNA), comprising up to 90% of the total RNA.</p></li>
<li><p>For especific isolation of mRNA transcripts, in addition to rRNA depletion, poly(A) must be done for selecting the RNAs containing a polyadenilated tail using oligo primers.</p></li>
</ul>
</div>
</section>
<section id="dna-library-bias">
<h2>DNA library bias<a class="headerlink" href="#dna-library-bias" title="Link to this heading"></a></h2>
<p>Depending on template, DNA-Seq can include Whole Genome Sequencing (WGS), Whole Exome Sequencing (WES),, Epigenome Sequencing (ChIP-Seq, WGBS) or Targeted Sequencing (TS).
Among the different library preparation steps presented earlier, several biases can be introduced during the process.
Here are presented the main biases introduced for DNA in each library preparation step and possible solutions to avoid them.</p>
<a class="reference internal image-reference" href="_images/library_prep_explanation_Van_Djik_2014.jpg"><img alt="*source: https://doi.org/10.1016/j.yexcr.2014.01.008*" class="align-center" src="_images/library_prep_explanation_Van_Djik_2014.jpg" style="width: 400px;" /></a>
<ul class="simple">
<li><p><strong>Fragmentation</strong></p></li>
</ul>
<p>Chromatin sonication for ChIP-seq has been shown to be non-random, with euchromatin being sheared more efficiently than heterochromatin.</p>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>To solve this it has been developed the double-fragmentation ChIP-seq protocol.</p>
</div>
<ul class="simple">
<li><p><strong>Size Selection</strong></p></li>
</ul>
<p>Agarose gel slices by heating to 50 ºC in chaotropic salt buffer decreased the representation of AT-rich sequences.</p>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Simple solution to this problem is to melt the gel slices in the supplied buffer at room temperature (18–22 ºC), considerably reducing GC bias.</p>
</div>
<ul class="simple">
<li><p><strong>PCR</strong></p></li>
</ul>
<p>Introduce bias in sample composition, due to the fact that not all fragments in the mixture are amplified with the same efficiency.
GC-neutral fragments are amplified more efficiently than GC-rich or AT-rich fragments, and as a result fragments with high AT- or GC content may become underrepresented or are completely lost during library preparation</p>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<ul class="simple">
<li><p>Ligate adapters that contain all necessary elements for bridge amplification on Illumina flowcells are preferred, eliminating the need for PCR to add these sequences afterwards. Nevertheless, requires relatively large quantities (41 mg) of input material.</p></li>
<li><p>In the extreme case of small input amount, the single cell,multiple displacement amplification (MDA) may be the preferred amplification method. MDA is an extremely powerful amplification method, allowing microgram quantities of DNA to be obtained from femtograms of starting material. For this reason, MDA has become the method of choice for whole genome amplification (WGA) from single cells</p></li>
<li><p>PCR additives have also been reported to reduce bias, such as betaine or tetramethylammonium chloride (TMAC) may help to further improve coverage of extremely GC-rich or AT-rich regions.</p></li>
<li><p>The best overall performing polymerase appears to be Kapa HiFi.</p></li>
</ul>
</div>
<div class="admonition seealso">
<p class="admonition-title">See also</p>
<p>For more information see the publication <a class="reference external" href="http://dx.doi.org/10.1016/j.yexcr.2014.01.008">Library preparation methods for next generation sequencing: Tone down the bias</a>.</p>
</div>
</section>
<section id="rna-library-bias">
<h2>RNA library bias<a class="headerlink" href="#rna-library-bias" title="Link to this heading"></a></h2>
<p>On this section are presented the main source of bias in RNA-seq, and the solutions that could be implemented to reduce it.</p>
<a class="reference internal image-reference" href="_images/protocol_RNA-seq_library_bias_vanDjik_etal_2014.png"><img alt="*source: https://doi.org/10.1016/j.yexcr.2014.01.008*" class="align-center" src="_images/protocol_RNA-seq_library_bias_vanDjik_etal_2014.png" style="width: 400px;" /></a>
<section id="sample-preservation-and-isolation">
<h3><strong>Sample Preservation and Isolation</strong><a class="headerlink" href="#sample-preservation-and-isolation" title="Link to this heading"></a></h3>
<ol class="arabic">
<li><p>Degradation of RNA:</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Minimizing the sample processing and freezing and thawing cycles, ensures that RNA is preserved as best as possible.</p>
</div>
</div></blockquote>
</li>
<li><p>RNA extraction:</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>If possible use high concentrations of RNA samples or avoid TRIzol extraction altogether.</p>
</div>
</div></blockquote>
</li>
</ol>
</section>
<section id="library-construction">
<h3><strong>Library Construction</strong><a class="headerlink" href="#library-construction" title="Link to this heading"></a></h3>
<ol class="arabic">
<li><p><strong>Low-quality and/or low-quantity RNA samples</strong>:</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>RNase H has been the best method for detecting low-qualityRNA and even could eﬀectively replace the standard RNA-seq method based on oligo (dT).
For low-quantity RNA,the SMART and NuGEN approaches had lower duplication rates and signiﬁcantly decreased the necessary amount of starting material compared to other methods.</p>
</div>
</div></blockquote>
</li>
<li><p><strong>mRNA enrichment bias</strong>: In eukaryotes enrich for polyadenylated RNA transcripts with oligo (dT) primers have shown that this method remove all non-poly (A) RNAs, such a reolication-dependant histones and lncRNAs (lacking of polyA),or incomplete mRNAs.</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Targeting rRNA as depletion method will not limit to only mRNA molecules, may capture more immature transcripts, leading to a complexity increase of sequencing data (also is more expensive).
Subtractive hybridization using rRNA-specific probes as the method that introduced the least bias in relative transcript abundance,</p>
</div>
</div></blockquote>
</li>
<li><p><strong>RNA fragmentation bias</strong>: There are two major approaches of RNA fragmentation: chemical (using metal ions) and enzymatic (using RNase III). During this process could be introduced lenght biases or errors (propagated to later cycles).</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Studies have shown that methods that involve non speciﬁc restriction endonucleases indicate less sequence bias and have been shown to perform similarly to the physical methods. Also enzymatic methoda are easy to automate</p>
</div>
</div></blockquote>
</li>
<li><p><strong>Primer bias</strong>: During reverse transcription into cDNA by random hexamers can lead to deviation of nucleotide content of RNA sequencing reads, resulting in low complexity of RNA sequencing data.</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Could be avoid using the Illumina Genome Analyzer, which perform the reverse transcription directly on the flowcells, avoiding the PCR.
Also has been proposed a bioinformatics tool in a reweighing scheme to adjust for the bias and make the distribution of the reads more uniform.</p>
</div>
</div></blockquote>
</li>
<li><p><strong>Adapter ligation bias</strong>: Adapter ligation introduces a significant but widely overlooked bias in the results of NGS small RNA sequencing.</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>As a solution, several groups propose to randomize the 3’ end of the 5’adapter and the 5’end of the 3’adapter.
The strategy is based on the hypothesis that a population of degenerate adapters would average out the sequencing bias because the slightly different adapter molecules would form stable secondary structures with a more diverse population of RNAsequences           - Reverse transcription bias: reverse transcriptases tend to produce false second strand cDNA throughDNA-dependent DNA polymerase. ActinomycinD, a compound that specifically inhibits DNA-dependent DNAsynthesis, has been proposed as an agent to eliminate antisense artifacts</p>
</div>
</div></blockquote>
</li>
<li><p><strong>Reverse Transcription</strong>: A known feature of reverse transcriptases is that they tend to produce false second strand cDNA through DNA-dependent DNA polymerase. This may not be able to distinguish the sense and antisense transcript and create difficulties for the data analysis.</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<ul class="simple">
<li><p>The deoxyuridine triphosphate (dUTP) method, one of the leading cDNA-based strategies, can be specifically removed by enzymatic digestion</p></li>
<li><p>Another method is to synthesize the first strand of cDNA using labeled random hexamer primer and SSS using DNA-RNA template-switching primer</p></li>
</ul>
</div>
</div></blockquote>
</li>
<li><p><strong>PCR amplification bias</strong>: main source of artifacts and base composition bias in the process of library construction:</p>
<blockquote>
<div><p>7.1. Extremely AT/GC-Rich: Fragments of GC-neutral can be ampliﬁed more than GC-rich or AT-rich fragments.</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<ul class="simple">
<li><p>Through the use of custom adapters, the samples without ampliﬁcation and ligation can be hybridized directly with the oligonucleotides on the ﬂowcell surface, thus avoiding the biases and duplicates of PCR.</p></li>
<li><p>However, the ampliﬁcation-free method requires high sample input, which limits its widely used. The most eﬀective PCR enhancing additives currently used are betaine.</p></li>
</ul>
<p>It is an amino acid mimic that acts to balance the diﬀerential T m between AT and GC base pairs and has been eﬀectively used to improve the coverage of GC-rich templates
- Presence of tetramethylammonium chloride (TMAC) showed that can remarkably increase the ampliﬁcation of AT-rich regions in Kapa HiFi in the presence. Additionally,
a number of additives have been reported to play an important role in reducing the bias of PCR ampli-ﬁcation, including small amides such as formamide, small sulfoxides such as dimethyl sulfoxide (DMSO),
or reducingcompounds such as β-mercaptoethanol or dithiothreitol(DTT).</p>
</div>
</div></blockquote>
<p>7.2. PCR cyle: PCR can exponentially amplify DNA/cDNA templates, thus leading to a signiﬁcant increase of ampliﬁcation bias with the number of PCR cycles.</p>
<blockquote>
<div><div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>it is recommended that PCR be performedusing as few cycle numbers as possible to mitigation bias.</p>
</div>
</div></blockquote>
</div></blockquote>
</li>
</ol>
<div class="admonition seealso">
<p class="admonition-title">See also</p>
<p>For more information see the publication <a class="reference external" href="http://dx.doi.org/10.1016/j.yexcr.2014.01.008">Library preparation methods for next generation sequencing Tone down the bias</a> and <a class="reference external" href="https://doi.org/10.1155/2021/6647597">Bias in RNA-seq Library Preparation: Current Challenges and Solutions</a>.</p>
</div>
</section>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="about.html" class="btn btn-neutral float-left" title="About the course" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="2-%20Sequencing%20technologies.html" class="btn btn-neutral float-right" title="2 Main Sequencing Technologies" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright .</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>