<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="./">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>2 Main Sequencing Technologies &mdash; NGS-QC-Course  documentation</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=80d5e7a1" />
      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=19f00094" />

  
    <link rel="canonical" href="https://fabianandradelozano.github.io/NGS-QC-Course/2- Sequencing technologies.html" />
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=5929fcd5"></script>
        <script src="_static/doctools.js?v=9a2dae69"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="author" title="About these documents" href="about.html" />
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            NGS-QC-Course
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
<li class="toctree-l1"><a class="reference internal" href="about.html">About the course</a></li>
<li class="toctree-l1"><a class="reference internal" href="3-%20Quality%20Control%20and%20Preprocessing.html">3 Quality Control and Preprocessing</a></li>
<li class="toctree-l1"><a class="reference internal" href="4-%20Quality%20of%20the%20mapping.html">4 Quality of the Mapping</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">NGS-QC-Course</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">2 Main Sequencing Technologies</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/2- Sequencing technologies.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="main-sequencing-technologies">
<span id="library-preparation-page"></span><h1>2 Main Sequencing Technologies<a class="headerlink" href="#main-sequencing-technologies" title="Link to this heading"></a></h1>
<section id="short-reads-sequencing-illumina">
<h2>Short Reads sequencing (Illumina)<a class="headerlink" href="#short-reads-sequencing-illumina" title="Link to this heading"></a></h2>
<p>It consist in The  polymerase-mediated Sequencing by synthesis (SBS), this works by coupling the four DNA bases to fluorescent markers alongside a terminator chemical group that pauses DNA synthesis.
While DNA is being synthesized, each fluorescent marker is optically verified before the tag and terminator are removed, and the next step in the sequence is recorded.</p>
<a class="reference internal image-reference" href="_images/illumina_Lu_et_al_2016.png"><img alt="*Source: https://www.researchgate.net/publication/357946568_New_approaches_and_concepts_to_study_complex_microbial_communities*" class="align-center" src="_images/illumina_Lu_et_al_2016.png" style="width: 400px;" /></a>
<p>#1. Cluster generation</p>
<p>Adapter attached to the DNA fragment is used to hybridisation to the flowcell, subsequentlty PCR amplification (bridge amplification) generates a cluster of the same sequence fragment to amplify the signal
when the nucleotide base is synthesized, thus obtaining a multiple cluster on a Flow Cell.</p>
<p>#2. Sequencing</p>
<p>On each cycle is incorporated one nucleotide to the template, it correspond to the read length (100 cycles equal to 100 bp read length).
In each incorportation is imaged the fluorescent signal that indicates the base incorporated, and the terminator is removed to start the next cycle.</p>
<p>During Library preparation adapters are added to the DNA fragments, which are used to hybridize the DNA to the flow cell, and also act as barcaodes to identify the sample, when multiples samples are pooled in the same run.
Depending on the type of sequencing, the adapater can be added to one end of the fragment (single end) or both ends (paired end).</p>
<a class="reference internal image-reference" href="_images/single_vs_pair_end.png"><img alt="_images/single_vs_pair_end.png" class="align-center" src="_images/single_vs_pair_end.png" style="width: 400px;" /></a>
<section id="single-end">
<h3>Single end<a class="headerlink" href="#single-end" title="Link to this heading"></a></h3>
<p>Correspond to the basis of SBS, where the nucleotides added to the template sequence is read from one end of the fragment.
It’s more simple and effcient, also a reduce number of steps during the library preparation is required.
However, the quality of nucleotides decreases as the sequencing process progresses, so the ends of the reads tend to have lower quality scores.</p>
</section>
<section id="paired-end">
<h3>Paired end<a class="headerlink" href="#paired-end" title="Link to this heading"></a></h3>
<p><em>source: https://systemsbiology.columbia.edu/genome-sequencing-defining-your-experiment#:~:text=Single%2Dend%20vs.&amp;text=In%20single%2Dend%20reading%2C%20the,opposite%20end%20of%20the%20fragment.</em></p>
<p>During library preparation  are incorporated sequencing primers binding site at both ends of the DNA fragments.
This allows to reading at one read, when it finishes this direction at the specified read lenght, then starts another round of reading from the opposite end of the fragment.</p>
<p>It improves:</p>
<ul class="simple">
<li><p>The confidence of the sequence read</p></li>
<li><p>The ability to identify the relative positions of various reads in the genome (much more efficient in resolve rearrangements such as insertions, deletions or inversions)</p></li>
<li><p>Can improve the assembly of repetitive regions.</p></li>
<li><p>On the ohter hand is more expensive and time-consuming (respect single end)</p></li>
</ul>
<div class="admonition seealso">
<p class="admonition-title">See also</p>
<blockquote>
<div><p>See the <a class="reference external" href="https://www.youtube.com/watch?v=fCd6B5HRaZ8">Illumina_sequencing_by_synthesis_workflow</a> video by Illumina to visualize the concepts of SBS.</p>
</div></blockquote>
</div>
<p>For more information</p>
</section>
</section>
<section id="long-read-sequencing-nanopore">
<h2>Long read sequencing (Nanopore)<a class="headerlink" href="#long-read-sequencing-nanopore" title="Link to this heading"></a></h2>
<p>Use flow cells which contain an array of tiny holes — nanopores (protein pore) — embedded in an electro-resistant membrane. Each nanopore corresponds to its own electrode connected
to a channel and sensor chip, which measures the electric current that flows through the nanopore. When a molecule passes through a nanopore, the current is disrupted
to produce a characteristic ‘squiggle’. The squiggle is then decoded using basecalling algorithms to determine the DNA or RNA sequence in real time.
In an electrolytic solution, a constant voltage is applied to produce an ionic current through the nanopore such that negatively charged single-stranded DNA or RNA molecules
are driven through the nanopore from the negatively charged ‘cis’ side to the positively charged ‘trans’ side. Translocation speed is controlled by a motor protein that ratchets
the nucleic acid molecule through the nanopore in a step-wise manner. Changes in the ionic current during translocation correspond to the nucleotide sequence present in the sensing
region and are decoded using computational algorithms, allowing real-time sequencing of single molecules.  In addition to controlling translocation speed, the motor protein has helicase activity,
enabling double-stranded DNA or RNA–DNA duplexes to be unwound into single-stranded molecules that pass through the nanopore.</p>
<a class="reference internal image-reference" href="_images/Nanopore_principle.png"><img alt="_images/Nanopore_principle.png" class="align-center" src="_images/Nanopore_principle.png" style="width: 400px;" /></a>
<p>A basecaller translates raw signals into DNA sequence data (FASTQ). The basecaller uses a neural network to predict the most likely DNA sequence based on the raw signal data.</p>
<div class="admonition seealso">
<p class="admonition-title">See also</p>
<p>See the <a class="reference external" href="https://www.youtube.com/watch?v=RcP85JHLmnI">Nanopore_sequencing_workflow</a> video by Oxford Nanopore Technologies to visualize the concepts of Nanopore sequencing.</p>
</div>
</section>
<section id="fastq-format-and-phred-quality-score">
<h2>FASTQ format and Phred quality score<a class="headerlink" href="#fastq-format-and-phred-quality-score" title="Link to this heading"></a></h2>
<p>The raw data generated by the sequencer is stored in FASTQ format, which contains the sequence of nucleotides and their corresponding quality scores.
It it’s divided in four lines:</p>
<blockquote>
<div><ol class="arabic simple">
<li><p>Sequence identifier: starts with ‘&#64;’ and contains information about the read. Such as the instrument, run ID, flow cell ID, lane, tile, x, y coordinates, and read number.</p></li>
<li><p>Sequence: the nucleotide sequence of the read.</p></li>
<li><p>Quality identifier: starts with ‘+’ and contains the same information as the sequence identifier. Or it may be empty and in some cases is used for metadata.</p></li>
<li><p>Quality scores: the Phred quality score for each base in the read. The Phred quality score is a measure of the quality of the base call, which is calculated as -10 * log10(P), where P is the probability of the base call being incorrect. The quality score is represented as an ASCII character, with a score of 0 represented by ‘!’, and a score of 41 represented by ‘J’. The higher the quality score, the more confident we are in the base call.</p></li>
</ol>
</div></blockquote>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>The &#64; symbol can not be used for count the number of reads, because it could also appear as a quality score symbol.</p>
</div>
<a class="reference internal image-reference" href="_images/fastq_format.png"><img alt="_images/fastq_format.png" class="align-center" src="_images/fastq_format.png" style="width: 400px;" /></a>
</section>
</section>


           </div>
          </div>
          <footer>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright .</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>